### DISCUSSION TITLE
아카이빙 관련 논의

### DISCUSSION THREAD

**Discussion by outsideris** [nodejs/nodejs-ko#1200] [General]
https://github.com/nodejs/nodejs-ko/issues/1198 이슈와 관련해서 현재 블로그를 번역하고 있는 i18n 팀은 없는 것 같고 crowdin으로 넘어가서 아카이빙하겠다고 합니다. 

최근까지도 여기서 활동하신 분도 있고 해서 논의를 해보기 위해서 디스커션을 열었습니다. 

cc @taggon  @yous @marocchino

**Comment by outsideris** [comment]
https://crowdin.com/project/nodejs-website
https://crowdin.com/project/nodejs

여기인거 같은데 여기도 딱히 액티브하진 않아보이는군요.

**Reply by yous** [reply]
https://crowdin.com/project/nodejs-website 쪽이 보통 저희가 하던 작업이 될 것 같은데, 내용을 보려면 일단 프로젝트에 join을 해야하는 것 같네요.

**Reply by marocchino** [reply]
저는 저 클라우드인 그룹 맴버인데 안보이는 걸 보면 뭔가 설정이 꼬여있는 것 같네요.
다른 사례도 보이고.. https://crowdin.com/project/nodejs-website/discussions/73

**Reply by yous** [reply]
그러네요. join 신청해서 들어왔는데도 아무것도 없는 것처럼 보입니다.

**Reply by yous** [reply]
이게 nodejs.org의 crowdin 설정 같은데요, https://github.com/nodejs/nodejs.org/blob/main/crowdin.yml

node/i18n 저장소는 api 문서화 쪽에 집중하고 있는 듯합니다. https://github.com/nodejs/i18n/blob/master/crowdin.yml

**Comment by outsideris** [comment]
논의만 만들어놓고 챙기질 못하고 있었네요. crowdin으로 가면 현재 같은 분위기는 나지 않을 것 같은데  i18n 현재 규칙 등을 좀 더 찾아보고 다시 오겠습니다.

**Comment by taggon** [comment]
대충 분위기 보니까, 기존 블로그 번역을 계속하는 로컬 그룹은 이제 없는 것 같습니다.
이미 다른 로컬 그룹의 저장소는 전부 아카이빙됐고, @nodejs/nodejs-ko 만 유일한 예외로 남아있네요. 그런데 딱히 릴리스 노트의 번역을 필요로 하는 거 같지도 않고, 우리가 하던 활동을 이어갈 방법도 보이지 않고요.

이쯤에서 로컬 그룹 활동을 마무리하는 건 어떨까하는 의견을 내어봅니다.

**Comment by outsideris** [comment]
오랫동안 했던거라 저도 고민을 좀 해봤는데 전처럼 활동이 활발하지도 않고 여기서 마무리 하는것도 좋아보입니다.
새 번역 프로세스는 관심있는 사람이 새 절차에 따라서 하면 될것 같습니다.

**Comment by yous** [comment]
저도 활동 마무리하는 쪽으로 기우는 것 같습니다. io.js 시절부터 해왔으니 상당히 오래되었고, 정보를 얻을 수 있는 다른 Node.js 커뮤니티가 많을 거라 생각도 됩니다.

**Comment by taggon** [comment]
그러면 제가 원래 글에다가 구성원들이 아카이빙에 동의했다고 남겨두겠습니다.
